feathr_project/feathr/client.py

```diff
@@ -946,19 +947,32 @@ def _collect_secrets(self, additional_secrets=[]):
             prop_and_value[prop] = self.envutils.get_environment_variable_with_default(prop)
         return prop_and_value
 
-    def get_features_from_registry(self, project_name: str) -> Dict[str, FeatureBase]:
+    def get_features_from_registry(self, project_name: str, return_keys: bool = False, verbose: bool = False) -> Union[Dict[str, FeatureBase], tuple]:
```

Comment:
return type is
`Union[Dict[str, FeatureBase], Tuple[Dict[str, FeatureBase], Dict[str, TypedKey]]]`.

Reply:
The TypedKey could also be a list of TypedKey

Reply:
Changed